package.json

```diff
@@ -95,6 +98,7 @@
   },
   "dependencies": {
     "@folio/react-intl-safe-html": "^3.1.0",
+    "eslint-import-resolver-webpack": "^0.13.2",
```

Comment:
Things related to eslint should be dev-deps since linting happens only in development, not in production.

Reply:
Yes, my bad. Will be done. Thanks!